wca-regulations.md

```diff
@@ -325,7 +325,15 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 11e1) If a competitor is granted an extra attempt, the extra attempt must be scrambled using a different scramble sequence. This scramble sequence must be generated using a current official version of an official WCA scramble program (see [Regulation 4f](regulations:regulation:4f)).
     - 11e2) If a competitor is granted an extra attempt, the extra attempt should be done right after the attempt that caused it, and must replace the original regularly numbered attempt.
     - 11e3) If it is unclear whether an incident should result in an extra attempt, the competitor may be granted a provisional extra attempt that will be used only if it is later found that an extra attempt was appropriate (e.g. by a decision of the WRC).
-- 11f) Decisions about an incident may be supported with video or photographic analysis, at the discretion of the WCA Delegate.
+- 11f) Decisions about an incident may be supported with video, photographic, or audio evidence, at the discretion of the WCA Delegate.
+    - 11f1) Video, and similar forms of recorded evidence, should be reviewed at full speed. Exception: slow-motion or frame-by-frame analysis may be used in the following circumstances:
+        - 11f1a) The individual result is a regional record or a personal record in the top 50 results of the world rankings.
+        - 11f1b) The average or mean the result is a part of is a regional record or a personal record in the top 50 results of the world rankings.
```

Comment:
```suggestion
        - 11f1b) The average or mean the result is a part of a regional record or a personal record in the top 50 results of the world rankings.
```

Reply:
We can match this wording with the misscrambles wording